### ISSUE TITLE
packge libintl-devel not exitst?

### ISSUE THREAD

**Issue by CCTV-1** [Alexpux/MSYS2-packages#1093]
Just now I tried using glib on msys2,but my code can not compile normally.
commad:
`pkg-config --cflags --libs glib-2.0`.
output:
`-I/usr/include/glib-2.0 -I/usr/lib/glib-2.0/include -lglib-2.0 -lintl -lpcre -lintl -liconv -lpcre
`.
compile command:
`gcc test-glib.c -Wall -Wextra -Wpedantic -std=gnu11 -O3 -m64 -I/usr/include/glib-2.0 -I/usr/lib/glib-2.0/include -lglib-2.0 -lintl -lpcre -lintl -liconv -lpcre -o test-glib`
but ld output:
`/usr/lib/gcc/x86_64-pc-msys/6.4.0/../../../../x86_64-pc-msys/bin/ld: cannot find -lintl`.
I use `pactree glib2-devel | grep devel` output follows not exitst `libintl-devel:`
```
glib2-devel
├─pcre-devel
├─libffi-devel
├─libiconv-devel
└─zlib-devel
```
Then I tried `pacman -Ss intl` output:
```
msys/intltool 0.51.0-2 (base-devel) [installed]
    The internationalization tool collection
msys/libintl 0.19.8.1-1 (libraries) [installed]
    GNU Internationalization runtime library
```
so now msys2 for libintl It is not yet planned build development packge or packaged in other packge?

**Comment by mati865** [comment]
Libintl is split package from gettext, install `gettext-devel` and it should work.

**Comment by CCTV-1** [comment]
@mati865 yes,compile success after installing `gettext-devel`,but why `glib2-devel` does not depend on `gettext-devel` or split `gettext-devel` to avoid this case?

**Comment by mati865** [comment]
It's a bug, glib-devel should depend on gettext-devel.

31.12.2017 01:32 "CCTV-1" <notifications@github.com> napisał(a):

**Comment by cbrt64** [comment]
Got here via an unrelated search for `pactree`. I'll just leave a note that this was finally fixed a year ago by #2313.
